geoportal/c2cgeoportal_geoportal/views/profile.py

```diff
@@ -64,12 +64,6 @@ def csv(self):
         result = _("distance") + "," + ",".join(layers) + ",x,y"
         template = ",".join("%s" for l in layers)
         for point in points:
-            # Handles cases when a layer is undefined, thus when not all raster
-            # have the same geographical coverage
-            for l in layers:
-                if l not in point["values"]:
-                    point["values"][l] = -9999
```

Comment:
This is wanted ...

Reply:
For the CSV the client needs -9999 for no data in the export, for the Json we do what we want, and I prefer null, it's clearer ...